x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/util/StringUtils.java

```diff
@@ -23,12 +23,16 @@
 
 import static java.util.stream.Collectors.toList;
 
-public abstract class StringUtils {
+public final class StringUtils {
+
+    private StringUtils() {}
 
     public static final String EMPTY = "";
     public static final String NEW_LINE = "\n";
     public static final String SQL_WILDCARD = "%";
 
+    private static final String[] INTEGER_ORDINALS = new String[] { "th", "st", "nd", "rd", "th", "th", "th", "th", "th", "th" };
```

Comment:
What's the first `th` for? 10th, 20th, 30th...?

Reply:
yep